src/components/Combobox/__tests__/__snapshots__/combobox-snapshot-tests.jest.js.snap

```diff
@@ -949,7 +949,7 @@ exports[`Combobox renders correctly with loading 1`] = `
                 title="loading"
               >
                 <svg
-                  className="circle-loader-spinner input-component__loader-svg"
+                  className="input-component__loader-svg circle-loader-spinner"
```

Comment:
if the component is the same but only the order of the classes changes I think it will be the best to undo the change (put it in the same order as before) for not breaking snapshot tests in dapules of developers using this loader.

Reply:
and I think this snap file is not updated?